#! /bin/sh /usr/share/dpatch/dpatch-run
## 60_desktop-entry.dpatch by Yavor Doganov <yavor@gnu.org>
##
## DP: Remove deprecated `Encoding' field and extension of the icon.
## DP: Fix the `Category field as per the current specification.  Add
## DP: `MimeType' field.

@DPATCH@
diff -urNad kazehakase-0.5.8~/data/kazehakase.desktop kazehakase-0.5.8/data/kazehakase.desktop
--- kazehakase-0.5.8~/data/kazehakase.desktop	2009-10-01 21:06:59.000000000 +0300
+++ kazehakase-0.5.8/data/kazehakase.desktop	2009-10-03 17:18:15.000000000 +0300
@@ -1,5 +1,4 @@
 [Desktop Entry]
-Encoding=UTF-8
 Name=Kazehakase Web Browser
 Name[ja]=風博士ウェブブラウザ
 Comment=Web browser
@@ -9,4 +8,5 @@
 Terminal=false
 Type=Application
 Icon=kazehakase-icon
-Categories=Application;Network;
+Categories=Network;WebBrowser;
+MimeType=text/html;text/xml;application/xhtml+xml;
